spec/controllers/api/service_providers_controller_spec.rb

```diff
@@ -6,8 +6,10 @@ def response_from_json
   end
 
   describe '#index' do
+    let(:team) { build(:team, agency: build(:agency)) }
+
     it 'returns active, approved SPs' do
-      sp = create(:service_provider, active: true, approved: true)
+      sp = create(:service_provider, active: true, approved: true, team: team)
```

Comment:
There is a `:with_team` trait you could use from the sp factory

Reply:
That's great, will use. Thanks!